void bug1830() {
    try {
        
    }
    catch (Exception exception)
}